server/models/article.js

```diff
@@ -38,6 +57,42 @@ articleSchema.statics.urisAreEqual = (uri1, uri2) => {
   return normalizeURI(uri1) === normalizeURI(uri2);
 };
 
+articleSchema.methods.getMercuryInfo = function getMercuryInfo() {
+  const encodedURI = encodeURIComponent(this.uri);
+  return fetch(`https://mercury.postlight.com/parser?url=${encodedURI}`,
+  { headers: { 'Content-Type': 'application/json', 'x-api-key': process.env.MERCURY_API_KEY } })
+  .then((res) => {
+    return res.json();
+  }).then((json) => {
+    if ( // Mercury error conditions:
```

Comment:
This is kind of weird cause it seems like Mercury doesn't always fail in the same way. I've seen 3 different types of error responses so far:
- just null
- { "message": null }
- { "error": true, "messages": <error message> }
So this is potentially something to keep an eye on in case there are any other possibilities.